samples/scaleout/distributor-upgrade/Core_6/Worker1/Program.cs

```diff
@@ -23,7 +24,7 @@ static void Main()
         endpointConfiguration.EnlistWithLegacyMSMQDistributor(
             masterNodeAddress: ConfigurationManager.AppSettings["DistributorAddress"],
             masterNodeControlAddress: ConfigurationManager.AppSettings["DistributorControlAddress"],
-            capacity: 10);
+            capacity: PushRuntimeSettings.Default.MaxConcurrency);
```

Comment:
I think it normally is only used in the satellite registration code. Not sure if it is the best choice but it is also responsible for defining the default max concurrency.